### PR TITLE
Support refactored DBXWebUIEnvWrapper (old DatabricksClient)

### PR DESCRIPTION
**Pull request by roi-granulate**
## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the relevant documentation.
- [ ] I have added tests for new logic.

### CODE REVIEW

File: gprofiler/main.py [hunk 1]
```diff
@@ -977,9 +977,12 @@ def main() -> None:
...
         # "databricks" will be the default name in case of failure with --databricks-job-name-as-service-name flag
         args.service_name = "databricks"
-        databricks_client = DatabricksClient(logger)
-        if databricks_client.job_name is not None:
-            args.service_name = f"databricks-{databricks_client.job_name}"
+        dbx_web_ui_wrapper = DBXWebUIEnvWrapper(logger)
+        if (
+            dbx_web_ui_wrapper is not None
+            and (service_suffix := get_name_from_metadata(dbx_web_ui_wrapper.all_props_dict)) is not None
+        ):
```
Comment (michelhe) [inline gprofiler/main.py:984]: This line is too long and unreadable, please split it

**Review by michelhe — CHANGES_REQUESTED** [review]
Fix the linting

File: gprofiler/main.py (hunk 1 above)
Comment (roi-granulate) [reply gprofiler/main.py]: 7342aaa

**Comment by roi-granulate** [comment]
@michelhe can you approve I can safely upgrade utils here? the only new commit will be [add auth method params ](https://github.com/Granulate/granulate-utils/commit/e3d13d348dd98556cc8455b75d5901b58952d24e)

**Review by michelhe — APPROVED** [review]

**Review by Jongy — APPROVED** [review]

**Pull request merged**